src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -182,6 +184,10 @@ class HaskellCompletionContributor extends CompletionContributor {
                 resultSet.addAllElements(findTopLevelTypeSignatureLookupElements(psiFile).asJavaCollection)
               }
 
+              if (HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
Ok, maybe another completionContributor for `Haskell` language, is that possible to have multiple completionContributor for a single language?